server/room-battle.ts

```diff
@@ -491,6 +491,8 @@ export class RoomBattle extends RoomGames.RoomGame {
 	turn: number;
 	rqid: number;
 	requestCount: number;
+	playerTeamResolver?: (args: string[]) => void;
+	resolveLogData?: (args: string[]) => void;
```

Comment:
```suggestion
	playerTeamResolver?: (args: string[]) => void;
	resolveLogData?: (args: string[]) => void;
```

To protect against race conditions, this should probably be a single array of requests.